lms/templates/verify_student/incourse_reverify.underscore

```diff
@@ -17,7 +17,7 @@
               <li class="help-item"><%- gettext( "Make sure your face is well-lit" ) %></li>
               <li class="help-item"><%- gettext( "Be sure your entire face is inside the frame" ) %></li>
               <li class="help-item">
-                <%=  HtmlUtils.interpolateHtml( gettext( "Once in position, use the camera button {icon} to capture your photo" ), { icon: HtmlUtils.HTML('<span class="example">(<span class="icon fa fa-camera" aria-hidden="true"></span>)</span>') } ) %>
+                <%=  edx.HtmlUtils.interpolateHtml( gettext( "Once in position, use the camera button {icon} to capture your photo" ), { icon: edx.HtmlUtils.HTML('<span class="example">(<span class="icon fa fa-camera" aria-hidden="true"></span>)</span>') } ) %>
```

Comment:
@efischer19 I'd prefer to see us change the caller to use ``HtmlUtils.template``. I think you just need to change this one line (and import the dependency):

https://github.com/edx/edx-platform/blob/master/lms/static/js/verify_student/views/incourse_reverify_view.js#L43